webClient/src/app/browser-panel/browser-panel.component.ts

```diff
@@ -56,25 +57,23 @@ class FileRow {
     './browser-panel.component.scss',
     '../../styles.scss'
     ]
+    
 })
-export class BrowserPanelComponent implements OnInit {
+export class BrowserPanelComponent implements AfterViewInit, OnInit {
     // @ts-ignore
     @Input() connection: Connection;
     // @ts-ignore
     @Input() ftaSide: FTASide;
 
-    fileView: string;
+    @ViewChild(ZluxFileExplorerComponent)
+    private fileExplorer: ZluxFileExplorerComponent;
 
```

Comment:
This is a bit anal, but if we're making this private, then the others should be public/private too or remove the private tag completely. Private/public doesn't mean much in Angular anyway, at run-time it's all the same.